src/components/pages/Animations/FaceoffReveal.js

```diff
@@ -277,7 +296,7 @@ const FaceoffReveal = props => {
           <animated.h1 className="points" style={countPointsStyle}>
             {countPointsStyle.number.interpolate(val => Math.floor(val))}
           </animated.h1>
-          <animated.h1 className="points" style={divAppearStyle2}>
+          <animated.h1 className="points padding" style={divAppearStyle2}>
```

Comment:
Let's make these h1 tags be p tags, since they are the "content" of this section, using the `h1` className